controllers/handlers/kubevirt.go

```diff
@@ -1036,3 +1034,10 @@ func getLabelPatch(dest, src map[string]string) ([]byte, error) {
 
 	return json.Marshal(patches)
 }
+
+func getKvPriorityClassName(hc *hcov1beta1.HyperConverged) string {
+	if hc.Spec.DefaultRuntimeClass != nil && *hc.Spec.DefaultRuntimeClass != "" {
+		return *hc.Spec.DefaultRuntimeClass
+	}
+	return kvPriorityClass
```

Comment:
This is probably too much.
By the way the idea of setting by default an higher priority class for VMs as part of the opinionated deployment is still debated.